hc_dpki/src/lib.rs

```diff
@@ -4,10 +4,11 @@ extern crate holochain_sodium;
 #[macro_use]
 extern crate arrayref;
 extern crate base64;
-extern crate rustc_serialize;
-
 extern crate bip39;
 extern crate boolinator;
+extern crate rustc_serialize;
```

Comment:
Why is this needed again?

Reply:
I don't know. I've just moved this by two lines for alphanumeric sorting.